packages/suite/src/views/settings/device/index.tsx

```diff
@@ -199,13 +217,32 @@ const Settings = () => {
                     <TextColumn
                         title={<Translation id="TR_FIRMWARE_VERSION" />}
                         description={
-                            <>
+                            <Version>
                                 <Translation
                                     id="TR_YOUR_CURRENT_FIRMWARE"
-                                    values={{ version: getFwVersion(device) }}
+                                    values={{
+                                        version: (
+                                            <VersionTooltip content={getFwRevision(device) || ''}>
+                                                <VersionLink
+                                                    target="_blank"
+                                                    href={`https://github.com/trezor/trezor-firmware/commit/${getFwRevision(
```

Comment:
move this link to https://github.com/trezor/trezor-suite/blob/develop/packages/suite/src/constants/suite/urls.ts

Reply:
The suite version url is not a constant as well. Would you want me to move it there as well?

https://github.com/trezor/trezor-suite/blob/9e8c13a5b2b666a2a06c7703fd3aa8e43dde67f9/packages/suite/src/views/settings/index.tsx#L364

Reply:
well... i guess it should be. 
@keraf is there any reason why `TrezorLink` component is not used there (in settings index)?
my guess is that TrezorLink component wasn't there yet

Reply:
The `TrezorLink` component was designed for `trezor.io` links and to be converted as Tor links when that feature is enabled. Not sure if there's a specific component for other links but it looks all good here for me.

Reply:
Okay so it sounds like I should move the url to the constants but keep the link the same?

Reply:
ok, so we have 4 different approaches here and i think it should be unified at some point
`TrezorLink` - for handling tor links
`ExternalLink` - which is using TrezorLink
'@trezor/components/Link' - used for external links, not trezor.io
`styled.a` - used for external links, not trezor.io

Reply:
14d766f

Reply:
great, thanks
i've addressed this link unification issue https://github.com/trezor/trezor-suite/issues/3633